recipes/gerrychain/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "gerrychain" %}
+{% set version = "0.2.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2d3dc1ff35c6f6c1670918f30e8be4c00faed1178b19501e38bfef09c2f023b5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - geopandas
+    - networkx
+    - pandas
+    - python
```

```suggestion
    - python >=3.6
```

I believe you use a python-3-ism in a few places.  It shows up in the build logs as syntax errors when compiling pyc files.